02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/DSS_Folder_Maker.md

```diff
@@ -0,0 +1,21 @@
+---
+aliases:DSS_Folder_Maker
+  -
+tags: 
+  - seedling
+publish: true
+---
+
+# DSS_Folder_Maker
+
+Available for: [[Windows Tools|Windows]]
+
+Available on: 
+
+  Obsidian Forum: <a href="https://forum.obsidian.md/t/ddc-automatic-folder-creation/31216"Obsidian Forum</a>
```

Comment:
This link gives me an "Oops this page doesn't exist.."

`https://forum.obsidian.md/t/ddc-automatic-folder-creation/31216"Obsidian Forum`

Does the `"` make it do something different in Obsidian?